src/utils/viewer/viewer.cpp

```diff
@@ -105,6 +120,16 @@ class Viewer : public Magnum::Platform::Application {
   bool drawObjectBBs = false;
 
   Magnum::Timeline timeline_;
+
+  // performance counter
+  uint64_t totalFrames_ = 0;
+  float timeDuration_ = 0.0f;
+  Cr::Containers::Pointer<Mn::Text::Renderer2D> performanceText_;
+  Mn::PluginManager::Manager<Mn::Text::AbstractFont> fontManager_;
```

Comment:
Bug: this is Cr::PluginManager.
I am wondering why the compiler never complain it.

Reply:
Because of https://github.com/mosra/magnum/blob/98232f383adc1ca1fdce877bc426951c23a89521/src/Magnum/Magnum.h#L40